### PR TITLE
Re-enable the automated tests

### PR DESCRIPTION
**Pull request by viorelaioia**
- Upgraded requirements versions
- Fixed stale imports
- Updated locators 
- Added a test for fxa login

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -1,32 +1,12 @@
...
-    --hash=sha256:a9e4f1398302e771ada4a0a268650815b43b354baf2de4e7e5023fb5f9675e50 \
-    --hash=sha256:0705803349964c7a2a144f1796a5d29905fe2a09931b2bb945ee0cb4deab75d7
-apipkg==1.4 \
-    --hash=sha256:65d2aa68b28e7d31233bb2ba8eb31cda40e4671f8ac2d6b241e358c9652a74b9 \
-    --hash=sha256:2e38399dbe842891fe85392601aab8f40a8f4cc5a9053c326de35a1cc0297ac6
-pytest-base-url==1.4.1 \
-    --hash=sha256:7425e8163345494ac7f544e99c6f3e5a08f4228bee5e26013b98c462a4d31f6e \
-    --hash=sha256:31e42366a5fc22f450b398837dc819bb7569f5e6bd5d74e494b2b9ec239876d1
+execnet==1.4.1
+py==1.8.0
```
Comment (akatsoulas) [inline requirements/test.txt]: Packages in the requirements need to include hashes. You can use `hashin` to get those. The easiest way to get it is by using pip, `pip install hashin`.

File: tests/functional/desktop/test_rewrites.py [hunk 2]
```diff
@@ -2,9 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+
 import pytest
 import requests
-import urllib
+import urllib.parse
```
Comment (akatsoulas) [inline tests/functional/desktop/test_rewrites.py]: No need to import parse. You can only import urllib and use parse by using the full path `urllib.parse`

File: requirements/test.txt (hunk 1 above)
Comment (viorelaioia) [reply requirements/test.txt]: Updated

**Comment by akatsoulas** [comment]
Closing this for now. We can revisit this in the future, when there will be some time to work on a complete CI/CD pipeline.

**Pull request closed without merging**